fortuna/src/marketplace/Marketplace.js

```diff
@@ -21,31 +70,25 @@ class Marketplace extends React.Component<{||}> {
 				</div>
 				<div className="row mt-5">
 		  			<div className="col-md-4">
-						<h2>Available Tanks</h2>
-						<h6>Tank One</h6>
-						<h6>Tank Two</h6>
-						<h6>Tank Three</h6>
-						<h6>Tank Four</h6>
-						<h6>Tank Five</h6>
-						<h6>Tank Six</h6>
-						<h6>Tank Seven</h6>
-						<h6>Tank Eight</h6>
-		  			</div>
-		  			<div className="col-md-4">
-						<h2 className="text-center">Selected Tank</h2>
+					  <div className="list-group col-md-6">
+						<a className="list-group-item list-group-item-action disabled">Category</a>
+						<a className="list-group-item list-group-item-action" onClick={() => this.showMarketView('CANNONS')}>Cannons</a>
+						<a className="list-group-item list-group-item-action" onClick={() => this.showMarketView('SCANNERS')}>Scanners</a>
+						<a className="list-group-item list-group-item-action" onClick={() => this.showMarketView('CHASSIS')}>Chassis</a>
+						<a className="list-group-item list-group-item-action" onClick={() => this.showMarketView('JAMMERS')}>Jammers</a>
+						<a className="list-group-item list-group-item-action" onClick={() => this.showMarketView('TREADS')}>Treads</a>
+						<a className="list-group-item list-group-item-action" onClick={() => this.showMarketView('ACCESSORIES')}>Accessories</a>
+						<a className="list-group-item list-group-item-action" onClick={() => this.showMarketView('CASUS_BLOCKS')}>Casus Blocks</a>
+					  </div>
 		  			</div>
 		  			<div className="col-md-4">
-						<h2 className="text-right">Components</h2>
-						<h6 className="text-right">Tank One Cannon</h6>
-						<h6 className="text-right">Tank One Tracks</h6>
-						<h6 className="text-right">Tank One Scanner</h6>
-						<h6 className="text-right">Tank One Jammer</h6>
-						<h6 className="text-right">Tank One Base</h6>
+						<div className="text-center">{this.state.marketplaceView}</div>
 		  			</div>
 				</div>
 	  		</div>
 		);
-  	}
+	  }
```

Comment:
I can't tell if this line used to have bad indentation or now has bad indentation, but it doesn't line up with the diff, so make sure it is right now.